bigbluebutton-html5/imports/ui/components/layout/layout-manager/camerasOnly.jsx

```diff
@@ -0,0 +1,387 @@
+
+    // do not proceed if using values from LayoutEngine
+    if (Object.keys(baseBounds).length > 0) {
+      return baseBounds;
+    }
+
+    const { navBarHeight } = DEFAULT_VALUES;
+
+    const cameraDockBounds = {};
+
+    const mobileCameraHeight = mediaAreaBounds.height * 0.7 - bannerAreaHeight();
+    const cameraHeight = mediaAreaBounds.height - bannerAreaHeight();
+
+    if (isMobile) {
+      cameraDockBounds.minHeight = mobileCameraHeight;
+      cameraDockBounds.height = mobileCameraHeight;
+      cameraDockBounds.maxHeight = mobileCameraHeight;
+    } else {
+      cameraDockBounds.minHeight = cameraHeight;
+      cameraDockBounds.height = cameraHeight;
+      cameraDockBounds.maxHeight = cameraHeight;
+    }
+
+    cameraDockBounds.top = navBarHeight + bannerAreaHeight();
+    cameraDockBounds.left = !isRTL ? mediaAreaBounds.left : null;
+    cameraDockBounds.right = isRTL ? sidebarSize : null;
+    cameraDockBounds.minWidth = mediaAreaBounds.width;
+    cameraDockBounds.width = mediaAreaBounds.width;
+    cameraDockBounds.maxWidth = mediaAreaBounds.width;
+    cameraDockBounds.zIndex = 1;
+
+    return cameraDockBounds;
+  };
+
+  const calculatesMediaBounds = () => {
+    const mediaBounds = {};
+    mediaBounds.width = 0;
+    mediaBounds.height = 0;
+    mediaBounds.top = 0;
+    mediaBounds.left = 0;
+
+    return mediaBounds;
+  };
+
+  const calculatesSidebarContentHeight = () => ({
+    minHeight: 0,
+    height: 0,
+    maxHeight: 0,
+  });
+
+  const calculatesLayout = () => {
+    const {
+      calculatesNavbarBounds,
+      calculatesActionbarBounds,
+      calculatesSidebarNavWidth,
+      calculatesSidebarNavHeight,
+      calculatesSidebarNavBounds,
+      calculatesSidebarContentWidth,
+      calculatesSidebarContentBounds,
+      calculatesMediaAreaBounds,
+      isTablet,
+    } = props;
+    const { captionsMargin } = DEFAULT_VALUES;
+
+    const sidebarNavWidth = calculatesSidebarNavWidth();
+    const sidebarNavHeight = calculatesSidebarNavHeight();
+    const sidebarContentWidth = calculatesSidebarContentWidth();
+    const sidebarNavBounds = calculatesSidebarNavBounds();
+    const sidebarContentBounds = calculatesSidebarContentBounds(sidebarNavWidth.width);
+    const mediaAreaBounds = calculatesMediaAreaBounds(
+      sidebarNavWidth.width,
+      sidebarContentWidth.width,
+    );
+    const navbarBounds = calculatesNavbarBounds(mediaAreaBounds);
+    const actionbarBounds = calculatesActionbarBounds(mediaAreaBounds);
+    const sidebarSize = sidebarContentWidth.width + sidebarNavWidth.width;
+    const cameraDockBounds = calculatesCameraDockBounds(mediaAreaBounds, sidebarSize);
+    const sidebarContentHeight = calculatesSidebarContentHeight();
+    const mediaBounds = calculatesMediaBounds(
+      mediaAreaBounds,
+      cameraDockBounds,
+      sidebarNavWidth.width,
+      sidebarContentWidth.width,
+      sidebarContentHeight.height,
+    );
```

Comment:
This function expects no arguments

```suggestion
    const mediaBounds = calculatesMediaBounds();
```